src/main/java/frc/robot/RobotContainer.java

```diff
@@ -22,23 +33,45 @@
  */
 public class RobotContainer {
 	// The robot's subsystems and commands are defined here...
-	public static final SwerveSubsystem swerve = new SwerveSubsystem();
-
-	public static final IMU imu = new IMU();
+	public static final SwerveSubsystem SWERVE = new SwerveSubsystem();
+	public static final Pigeon PIGEON = new Pigeon();
+	public static final PoseEstimator POSE_ESTIMATOR =
+			new PoseEstimator(SWERVE, PIGEON, new Pose2d(1, 1, PIGEON.getYawAsRotation2d()));
 
 	// Replace with CommandPS4Controller or CommandJoystick if needed
-	private final LimelightInterface limelightInterface = new LimelightInterface();
-	public static final CommandXboxController driveController =
-			new CommandXboxController(OperatorConstants.kDriverControllerPort);
+	private final LimelightInterface LIMELIGHT_INTERFACE = new LimelightInterface();
+
+	SendableChooser<Command> pathChooser = new SendableChooser<>();
+
+	//Constants for path following: DO NOT TOUCH THIS WITHOUT LETTING ME KNOW
```

Comment:
These are not Constants

Reply:
I accidentally used constants, but I meant to say that they were important variables.